src/main/java/com/ericsson/ei/controller/FreeController.java

```diff
@@ -0,0 +1,39 @@
+
+package com.ericsson.ei.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * No description
+ * (Generated with springmvc-raml-parser v.0.10.11)
+ * 
+ */
+@RestController
+@RequestMapping(value = "/query/free", produces = "application/json")
```

Comment:
why /query/free? just /query is enough

Reply:
It couldn't be /query, because when RAML generating interface for /query they will be common for /aggregatedObject, /missedNotifications and /query. Currently we have different interfaces for different endpoints, if just use /query will only one interface for all of this endpoints. Should be some other endpoint like /query/free or /freequery.